link/kprobe.go

```
@@ -191,6 +198,20 @@ func kprobe(symbol string, prog *ebpf.Program, opts *KprobeOptions, ret bool) (*
 	}
 
 	if opts != nil {
+		if opts.RetprobeMaxActive < 0 {
```

I think we shouldn't check this here but let the kernel fail if it is invalid. Alternative is `uint` as above.

---

Maybe we should let it pass to the kernel and let kernel set to a default value.

---

Agreed, let it be int and the kernel can deal with it.